#### Mobiman Framework Parameters

### Simulator:
# gazebo
# igibson
sim: igibson

flag_gazebo: true
flag_robot: true
flag_conveyor: false 
flag_bin: false
flag_pedsim: false
flag_moveit: false

### Rviz:
flag_rviz: true
rviz_config_path: "rviz/igibson/mobile_manipulator_ocs2_igibson"

### Map Server:
flag_map_server: true
config_map_server: "config_map_server"

### Robot:
flag_ns: true
robot_name: "jackalJaco"
n_robot: 2
robot_frame_name: "base_link"
urdf_path: "urdf/jackal_jaco.urdf"
urdf_path_ocs2: "urdf/jackal_fixedWheel_jaco_fixedFinger.urdf"    ## NUA NOTE: DEPRECATE THIS!
lib_path: "codegen/auto_generated/jackalJaco"
collision_points_config_path: "config/config_points_on_robot"

### Goal Server:
flag_goal_server: true

### Mobiman:
flag_mobiman: true
task_config_path: config/task/task_jackalJaco_igibson.info